app/grandchallenge/reader_studies/views.py

```diff
@@ -398,6 +398,27 @@ def generate_hanging_list(self, request, pk=None):
         )
         return Response({"status": "Hanging list generated."},)
 
+    @action(detail=True, methods=["patch"])
+    def remove_image(self, request, pk=None):
```

Comment:
What are the permissions on this endpoint?

Reply:
You need edit rights. Added a test for this.